inference-engine/src/mkldnn_plugin/mkldnn_memory.cpp

```diff
@@ -808,135 +698,14 @@ size_t MKLDNNMemoryDesc::getElementOffset(size_t elemNumber) const {
 }
 
 bool MKLDNNMemoryDesc::isCompatible(const MemoryDesc &rhs) const {
-    if (MemoryDescType::Blocked == rhs.getType()) {
-        return isCompatible(*(rhs.as<BlockedMemoryDesc>()));
-    } else if (MemoryDescType::Mkldnn == rhs.getType()) {
-        return isCompatible(*(rhs.as<MKLDNNMemoryDesc>()));
+    if (MemoryDescType::Mkldnn == rhs.getType()) {
+        return this->getShape() == rhs.getShape() && this->getPrecision() == rhs.getPrecision() &&
```

Comment:
If we really need to check shapes and precisions (the letter is really strange since `getPrecision` is a pure method, we have to move this details to the proper specialization.

Reply:
it was some misunderstanding from my side, when i thought that we will be support dynamic shapes for not convertible to ```oneDnnDesc``` descs.
so i removed this checks